### PR TITLE
OCPBUGS-6682: Switch from ifconfig to iproute2

### PR DESCRIPTION
**Pull request by mccv1r0**
Switch from ifconfig to iproute2

Fixes:  https://issues.redhat.com/browse/OCPBUGS-6682

Signed-off-by: Michael Cambria <mcambria@redhat.com>

**- What I did**

Switched command from ifconfig to iproute2 version 

**- How to verify it**

Command is logged when vSphere system boots OVNK 

**- Description for the changelog**

Change command used from ifconfig to iproute2 to plan for ifconfig deprecation and future RHEL9 support

### CODE REVIEW

**Comment by rbbratta** [comment]
could we also remove the `REMOVEME`, or is there a PR for that?

```
REMOVEME: Once BZ:1854355 is fixed, this needs to get removed.
```

**Comment by rbbratta** [comment]
or maybe reword to `DON'T REMOVE ME` ?  :smile:

**Comment by mccv1r0** [comment]
> or maybe reword to `DON'T REMOVE ME` ? smile

Does something like this work?

       # if the interface is of type vmxnet3 add multicast capability for that driver
-      # REMOVEME: Once BZ:1854355 is fixed, this needs to get removed.
+      # History: BZ:1854355 
       function configure_driver_options {

**Comment by yuqi-zhang** [comment]
is there someone from the networking side that could help verify/lgtm?

**Review by sinnykumari — APPROVED** [review]
